src/libraries/Microsoft.Extensions.Http/src/DefaultHttpClientFactory.cs

```diff
@@ -15,7 +16,7 @@
 
 namespace Microsoft.Extensions.Http
 {
-    internal class DefaultHttpClientFactory : IHttpClientFactory, IHttpMessageHandlerFactory
+    internal class DefaultHttpClientFactory : IHttpClientFactory, IHttpMessageHandlerFactory, IDisposable
```

Comment:
While you're here, can you seal this? Or it needs to stay unsealed for tests?

Reply:
Yeah, it's needed for tests :( 
https://github.com/dotnet/runtime/blob/3ad696ce0e20bf380af33117b35164a4b7bae47b/src/libraries/Microsoft.Extensions.Http/tests/Microsoft.Extensions.Http.Tests/DefaultHttpClientFactoryTest.cs#L511

Reply:
It'd be nice to clean that up subsequently.